src/js/inverted-index.js

```diff
@@ -1,233 +1,238 @@
 class InvertedIndex {
 
   /**
-     * Create an Inverted Index.
-     * @param {string} path - path to file.
-     */
-  constructor () {
+   * Create an Inverted Index.
+   *
+   * @param {Object} Indexes - holds each index as value and key as the index's file name.
+   */
+  constructor() {
     this.indexes = {}; // holds all indexes created
     this.path = require('path');
   }
+
   /**
-     * Read a File.
-     * @param {string} path - path to file.
-     * @return {string} fileData - read from a file.
-     */
-  readFile (filepath) {
+   * getJson
+   *
+   * Convert data to JSON Object.
+   *
+   * @param {string} readData - read from a file in byte or other form.
+   * @returns {Object}  - json object from the file.
+   */
+  readFile(filepath) {
     if (filepath) {
-      const file = filepath
-      const fs = require('fs')
+      const fs = require('fs');
       try {
-        let fileData = fs.readFileSync(filepath, 'utf-8')
+        const fileData = fs.readFileSync(filepath, 'utf-8');
         return fileData;
-      } catch(err) {
+      } catch (err) {
         if (err.code === 'ENOENT') {
-          return 'file Not Found'
+          return 'file Not Found';
         }
       }
-    }}
-
-  // readerFile() {
-    //       let reader = new FileReader()
-    //       reader.readAsText(this.path)
-    //       reader.onload = () => {
-    //           let json = JSON.parse(reader.result)
-    //           return this.json = json
-    //       }
-    //   }
+    }
+  }
 
   /**
-     * Convert data to JSON.
-     * @param {string} readData - read from a file in byte or other form.
-     * @return {Object} json object from the file.
-     */
-  getJson (readData) {
+   * getJson
+   *
+   * Convert data to JSON Object.
+   *
+   * @param {string} readData - read from a file in byte or other form.
+   * @returns {Object}  - json object from the file.
+   */
+  getJson(readData) {
     try {
-      return JSON.parse(readData)
+      return JSON.parse(readData);
     } catch (err) {
-      if (err instanceof SyntaxError)
-        return 'File Empty or Invalid Json object'
+      if (err instanceof SyntaxError) {
+        return false;
+      }
     }
-    return err
   }
 
-  /**
-   * Verify data is valid.
+  /*
+   * validateJsonData
+   *
+   * Verify data is valid and has both 'text' and 'title' as keys for each object.
+   *
    * @param {Object} JsonData - JSON read from a json file.
-   * @return {bool} - true || false.
+   * @returns {bool} - true || false.
    */
-  validateJsonData (JsonData) {
-    let dataValid = false
-    let keyArray = [] // hold all the objects keys
-    if (Array.isArray(JsonData)){
+  validateJsonData(JsonData) {
```

Comment:
@andela-Bgathu convention for naming variables in Javascript is `camelCase`, refactor.